Bottom: 77a17975f7b8d58a273f7dfe815fcf773c1c80a3
Top:    a48f507dcf8b1ff9e2531748094eb34ab9379655
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:47:30 -0500

Support asset host


---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index cfb91cca..a8da05f3 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -9,6 +9,11 @@
     - buildbot
     - buildbot-master
   tasks:
+  - add_host:
+      name: nexus # required. The hostname/ip of the host to add to the inventory, can include a colon and a port number.
+      groups: services # not required. The groups to add the hostname to, comma separated.
+      ip_addr: "{{ nexus_ip_addr }}"
+      uri: "http://{{nexus_hostname}}/repository"
   - name: Scan Gitlab ssh keys
     command: ssh-keyscan {{gitlab_hostname}}
     register: gitlab_keys
